Modules/_cursesmodule.c

```diff
@@ -5118,6 +5173,20 @@ cursesmodule_exec(PyObject *module)
     return 0;
 }
 
+/* Initialization function for the module */
+
+static struct PyModuleDef _cursesmodule = {
```

Comment:
It doesn't seem to be used.

Reply:
Actually, it's used in the `PyInit_Function`. I just moved it after the C API functions (just to be closer to where it's being used).

Reply:
Oh right, ignore my comment. I was confused by the diff in the PR.